test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3501,6 +3501,21 @@ def forward(self, x):
         x = torch.randn(1, 3, 2)
         self.run_test(GatherModule(), (x,))
 
+        class MyModule(torch.nn.Module):
+            def __init__(self):
+                super(MyModule, self).__init__()
+                self.register_buffer("rb", torch.randn(1, 1, 3, 1, 1))
+
+            def forward(self, x):
+                x += self.rb[0]
+                return x
+
+        x = torch.randn(1, 3, 224, 224)
+        self.run_test(MyModule(), (x,),
+                      dynamic_axes={"input": {0: "batch", 2: "height", 3: "width"},
+                                    "output": {0: "batch", 1: "class", 2: "height", 3: "width"}},
+                      input_names=['input'], output_names=['output'])
+
```

Comment:
Seems we don't need limit the `test_gather_constant_fold` test opsetVersion to cover more opset version tests?
Cause the 'Gather' op is merged during constant folding.

Reply:
good catch!